metrics/sinks/influxdb/influxdb.go

```diff
@@ -257,17 +257,48 @@ func (sink *influxdbSink) createDatabase() error {
 	if sink.dbExists {
 		return nil
 	}
+
+	//query databases and check the specific database is exist or not.
```

Comment:
`s/check the specific database is exist or not/check whether the specific database exist or not`